Flask_app/__init__.py

```diff
@@ -1,19 +1,26 @@
 import os
-from flask import Flask 
+
+from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
-from .config import config
 
- 
+from Flask_app.config import config
```

Comment:
Directory name should be capital and each word should be separated by underscore